packages/backend-app-api/src/wiring/BackendInitializer.test.ts

```diff
@@ -321,7 +321,7 @@ describe('BackendInitializer', () => {
       })(),
     );
     await expect(init.start()).rejects.toThrow(
-      "Extension point registered for plugin 'test-a' may not be used by module for plugin 'test-b'",
+      "Extension point 'a' registered for plugin 'test-a' may not be used by module 'mod' for plugin 'test-b'",
```

Comment:
I wonder if it might be even more readable to change which one is the "active" part. Something like

`Module 'mod' (made for plugin 'test-b') attempted to depend on extension point 'a' (made for plugin 'test-a') which is not possible because the plugin IDs do not match`

Or so? You get the culprit up front so you immediately know what to look into.

Reply:
sounds good 👍

Reply:
done